pkg/logs/client/http/destination_test.go

```diff
@@ -494,3 +494,34 @@ func TestTransportProtocol_HTTP1FallBack(t *testing.T) {
 	// Assert that the server automatically falls back to HTTP/1.1
 	assert.Equal(t, "HTTP/1.1", resp.Proto)
 }
+
+func TestTransportProtocol_HTTP1WhenUsingProxy(t *testing.T) {
+	c := configmock.New(t)
+
+	// Force client to use ALNP
+	c.SetWithoutSource("logs_config.http_protocol", "auto")
+	c.SetWithoutSource("skip_ssl_validation", true)
```

Comment:
You're sure the `skip_ssl_validation` is necessary? (I didn't check, I just find it surprising)

Reply:
Yeah the test server enforces TLS (I assume somewhere under the hood with an unverified certificate). TLS is required to validate HTTP/2, so we have to disable validation for the test to work.